changelogs/fragments/ios_facts_memorycalc.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
This would be a major change, as it impacts all the user directly with this change.
I am wondering if we can keep both somehow.

Moreover, FYI this PR would be held off until we plan a major release for the collection.
The target is we do major releases twice a year.
Regards

Reply:
@KB-perByte Are you proposing creating a new fact with the proper calculation and deprecating the incorrect fact?

Reply:
@pcunning-rh Yes that is the right thing to do. If we don't want to introduce a breaking change and for the older facts when the attributes are present a warning can be raised for the same.
Regards